apps/src/gamelab/AnimationTab/AnimationListItem.jsx

```diff
@@ -93,15 +94,26 @@ const AnimationListItem = React.createClass({
     style: React.PropTypes.object,
   },
 
+  getInitialState: function () {
+    return {
+      frameDelay: this.props.animationList.propsByKey[this.props.animationKey].frameDelay,
+      name: this.props.animationList.propsByKey[this.props.animationKey].name,
+      isNameValid: true
```

Comment:
Could use destructuring assignment to make some of these property chains shorter.  Even more so if you extract a helper for `this.props.animationList.propsByKey[this.props.animationKey]` which is very common in this file now.

```javascript
getAnimationProps() {
  const {animationList, animationKey} = this.props;
  return animationList.propsByKey[animationKey];
},

getInitialState() {
  const {name, frameDelay} = this.getAnimationProps();
  return {
    name,
    frameDelay,
    isNameValid: true
  };
}
```